src/scripts.js

```diff
@@ -78,33 +85,89 @@ const parseData = (data) => {
   //below functions will move to domManip on click events
   session.getTotalDollarSpentThisYear()
   session.getTripCost('Cartagena, Colombia', 7, 2)
+  addDestinationSearch()
   // session.getPastTrips(todayDate)
-  changeFormView()
+  // changeFormView(session, todayDate)
+  // displayPastTrips()
   // session.getUpcomingTrips(todayDate)
   // toggleView()
   // renderDom(data)
 }
 
+
+
 const changeFormView = () => {
   switch (tripView.value) {
   case 'past':
-    session.getPastTrips(todayDate)
+
+    let past = session.getPastTrips(todayDate)
+    displayTrips(past, tripView.value)
     break;
   case 'upcoming':
-    session.getUpcomingTrips(todayDate)
-
+    let upcoming = session.getUpcomingTrips(todayDate)
+    displayTrips(upcoming, tripView.value)
     break;
   case 'pending':
-    session.getPendingTrips()
+    let pending = session.getPendingTrips()
+    displayTrips(pending, tripView.value)
     break;
-  //   case 'expenses.':
-  //
-  //     break;
+    // case 'expenses.':
+    //
+    //   break;
   default:
     console.log('something went wrong');
```

Comment:
I'm assuming you're going to have this display on your dashboard as a user error?